docs/content/introduction.md

```diff
@@ -119,12 +119,14 @@ in defining responses to different situations, such as deviations from
 service-level objectives.
 
 When defining a policy, these are some key components to consider:
-[**Control Point**](./concepts/flow-control/selector.md) → Needed to guide flow
-control based decisions. [**Selector**](./concepts/flow-control/selector.md) →
-Needed to select the flow that the policy should consider.
-[**Classifier**](./concepts/flow-control/resources/classifier.md). → Create
-additional Flow Labels based on request metadata, collectively enriching the
-policy's operational capability.
+
+- [**Control Point**](./concepts/flow-control/selector.md): Needed to guide flow
+  control based decisions.
+- [**Selector**](./concepts/flow-control/selector.md): Needed to select the flow
+  that the policy should consider.
+- [**Classifier**](./concepts/flow-control/resources/classifier.md): Create
+  additional Flow Labels based on request metadata, collectively enriching the
+  policy's operational capability.
```

Comment:
@karansohi  Can you help me understand the reasoning behind putting this here? Also, we could be talking about it in your first policy doc.

Reply:
I was just fixing an indentation but yeah I agree it can be removed. Right? @tanveergill